framework/src/controller/README.md

```diff
@@ -1,10 +1,5 @@
 # Controller
 
-## Description
-
-The controller is a parent process, that is responsible for managing every user interaction with each [component](../components/README.md) and [module](../modules/README.md) of the framework.
-E.g. restarting the node, launching a rebuild verification process, etc.
-
 The controller is responsible for initialization of the infrastructure-level components.
```

Comment:
I think this
> The controller is responsible for initialization of the infrastructure-level components.
is not true.


Controller does not initialize components.
Is this correct? @nazarhussain

Reply:
Yes that's partially correct. But here I guess its not referring those components, would be better to rephrase it. 

```suggestion
The controller is responsible for initialization the communication bus and any other dependencies required to load the modules. If any module is configured to load as a child process, then it is the controller's responsibility to do so.
```